tests/unit_tests/toporder/test_nogam.py

```diff
@@ -0,0 +1,118 @@
+import networkx as nx
+import numpy as np
+import pytest
+
+from dodiscover import make_context
+from dodiscover.metrics import structure_hamming_dist, toporder_divergence
+from dodiscover.toporder.nogam import NoGAM
+from dodiscover.toporder.utils import dummy_groundtruth, dummy_sample, full_dag, orders_consistency
+
+
+@pytest.fixture
+def seed():
+    return 42
```

Comment:
You can just set seed in the test file as `seed = 42` or something. Unless there's a need to run these unit tests over multiple seeds?

Reply:
No, no need of multiple seeds